### ISSUE TITLE
App Configuration and App Protection bugs

### ISSUE THREAD

**Issue by zlDev** [MicrosoftDocs/OfficeDocs-Exchange#1472]
Hi MS Team

Noticed a couple of bugs, three bugs to be precise.

https://docs.microsoft.com/en-us/exchange/clients-and-mobile-in-exchange-online/outlook-for-ios-and-android/outlook-for-ios-and-android-configuration-with-microsoft-intune

Step 11.

"Mail Notifications" option is NOT available when I try to create a new App Configuration Policy. Only the "Calendar Notification" option is available however I can only set it to "Not Configured" or "Allowed". Whereas in the documentation available values are "Not Configured", "YES", "NO". See Screenshot below

![image](https://user-images.githubusercontent.com/8972968/67687950-fd318a80-f990-11e9-88bc-80b5e0a0f709.png)

If I look up the "Calendar Notification" option in Intune and click on the information bubble it says: Allow calendar notifications to display full details when the App Protection Policy setting "Org data notifications" is set to "Block Org Data".

There is no "Org data notifications" option in App Protection Policy. Neither this option is referenced anywhere in the documentation. See screenshots below

![image](https://user-images.githubusercontent.com/8972968/67684750-aa090900-f98b-11e9-8771-05e4c15ba022.png)

To summarise there are 3 issues here.
1. "Mail Notifications" option is NOT available in the App Configuration policy.
2. "Calendar Notification" option is available however you cannot select the correct values for it.
3. "Org data notifications" option does not exist in the App Protection Policy

Thanks

**Comment by scanum** [comment]
@zlDev Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by dariomws306** [comment]
Hi @zlDev, thank you for your feedback.

Let me double check this. I have verified with my demo tenant and I even don't have the "Calendar Notifications" option.

**Comment by dariomws306** [comment]
@mattpennathe3rd and @v-mapenn, could you help on this?

**Comment by zlDev** [comment]
Hi All

Can you provide an update on when the "Org data notifications" option will be rolled out ? New version of the outlook will support this from the 16th December.

**Comment by dariomws306** [comment]
Hi @zlDev,

Sorry but I don't have this information.

**Comment by scanum** [comment]
@zlDev thank you for submitting feedback.
I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome let me know if it is something that can be called out in the docs.
Please follow this link to contact support for business products:https://docs.microsoft.com/office365/admin/contact-support-for-business-products
Please keep us posted here on the resolution so we can feed whatever information you discover into the content.
